lms/djangoapps/instructor/views/api.py

```diff
@@ -2584,6 +2585,57 @@ def spoc_gradebook(request, course_id):
 
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_level('staff')
+def get_blank_lti(request, course_id):  # pylint: disable=W0613
+    """
+    Respond with 4-column CSV output of ID, email, grade (blank), max_grade (blank), and comments (blank)
+    """
+    course_id = SlashSeparatedCourseKey.from_deprecated_string(course_id)
+
+    def csv_response(filename, header, rows):
```

Comment:
Use the helper function https://github.com/Stanford-Online/edx-platform/blob/master/lms/djangoapps/instructor_analytics/csvs.py#L11

only difference is the header is not encoded, so just need to pass the header to the helper function already encoded